.travis.yml

```diff
@@ -57,6 +57,7 @@ script:
 after_success:
   # let's update the documentation and locales
   - if [ ${TRAVIS_PHP_VERSION:0:3} == "$CS" ] && [ "$GLPI_BRANCH" = "$AFTER_SUCCESS_BRANCH" ]; then tests/after_success.sh; fi
+  - if [ ${TRAVIS_PHP_VERSION:0:3} == "7.1" ] && [ "$GLPI_BRANCH" = "$AFTER_SUCCESS_BRANCH" ]; then tests/apidocs.sh; fi
```

Comment:
prefer a variable like we do with $CS, this is more maintainable

Reply:
This is the only place used for check the PHP 7.1 version, is really necessary?

Reply:
this is for convenience, when php will raise its version. This also gives the opportunity to comment the variable and tell it must be different from $CS to ensure apigen does not conflicts with other dependencies in composer.json